src/Components/SmartComponents/Remediation/Remediation.js

```diff
@@ -14,7 +14,10 @@ import { FormattedMessage } from 'react-intl';
 class Remediation extends Component {
     static propTypes = {
         systemId: propTypes.string,
-        selectedCves: propTypes.object,
+        selectedCves: propTypes.oneOfType([
```

Comment:
that's an interesting one! In which cases we send an array and in which an object? Can we unify by sending one type?

Reply:
For single CVE it's object for multiple it's an array. Yes, we could do Array.from in parent components.

Reply:
Well actually I think we should keep it one of Object/Array because purpose of propTypes is to say which type of prop can this component handle. And this component can handle both object (for single CVE) and array of objects (multiple CVEs). PropTypes should adapt to component not component to PropTypes.